frontend/app/services/cardService.ts

```diff
@@ -5,6 +5,7 @@ const IDIOMS_BACKEND_URL = process.env.EXPO_PUBLIC_IDIOMS_BACKEND_URL;
 
 const API_ROUTES = {
   IDIOMS: 'idioms/',
+  SMILEYS: 'smileys/', // Nueva ruta para los smileys
```

Comment:
Don't need this, please review the PR yourself before requesting a review to remove these comments